Networking/Networking/ApplicationPassword/ApplicationPasswordRequestProcessor.swift

```diff
@@ -3,29 +3,29 @@ import Foundation
 
 /// Authenticates and retries requests
 ///
-final class RequestProcessor {
+final class ApplicationPasswordRequestProcessor {
```

Comment:
The processor triggers generating application passwords when retrying requests, so it is specific to application password authentication only. I renamed this to avoid confusion with any other authentication method.

Reply:
I am afraid that using ApplicationPassword as a prefix wouldn't make sense here. 

The `RequestProcessor` handles authentication for both WPCOM token and application password cases. `RequestProcessor` uses `DefaultRequestAuthenticator` to perform the authentication [here](https://github.com/woocommerce/woocommerce-ios/pull/8659/files#diff-0dd78373a4e00f1b9699ccc252c7f856d7cf5250070c4ad57bc8e4d4921919b5R20-R24)


Extra info: Only REST requests are retried, and errors from other requests are just passed through. Due to [this method in `AlamofireNetwork `](https://github.com/woocommerce/woocommerce-ios/blob/trunk/Networking/Networking/Network/AlamofireNetwork.swift#L138-L143)